frontend/src/app/globals.css

```
@@ -3,5 +3,40 @@
 @tailwind utilities;
 
 body {
-  @apply bg-slate-100 text-slate-900 antialiased;
+  @apply bg-white text-stone-800 antialiased;
+  font-feature-settings: "cv02", "cv03";
```

Stylelint (17.14.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)